content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,74 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myconfigmap
+data:
+  username: "k8s-admin"
+  access_level: "1"
+```
+
+The following command will create the ConfigMap object:
+
+```shell
+kubectl apply -f myconfigmap.yaml
+```
+The output is similar to this:
+```console
+configmap/myconfigmap created
+```
```

Comment:
This kind of outputs are unnecessary. We sometimes include the output from a command because we have something to say about the output.